spec.html

```diff
@@ -4353,17 +4371,17 @@ <h1>GetValue ( _V_ )</h1>
         <p>The abstract operation GetValue takes argument _V_. It performs the following steps when called:</p>
         <emu-alg>
           1. ReturnIfAbrupt(_V_).
-          1. If Type(_V_) is not Reference, return _V_.
-          1. Let _base_ be GetBase(_V_).
+          1. If _V_ is not a Reference Record, return _V_.
           1. If IsUnresolvableReference(_V_) is *true*, throw a *ReferenceError* exception.
+          1. Let _base_ be _V_.[[Base]].
           1. If IsPropertyReference(_V_) is *true*, then
             1. If HasPrimitiveBase(_V_) is *true*, then
               1. Assert: In this case, _base_ will never be *undefined* or *null*.
```

Comment:
Since the `[[Base]]` field never holds `*undefined*` or `*null*`, I'm not sure this is very useful as worded. Also, it fails to exclude environment records and `~empty~`. I suggest changing this to something like

```suggestion
              1. Assert: _base_ is an ECMAScript language value other than *undefined* or *null*.
```

Reply:
Nice, changed in d8f4575.